.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+## Bug Description
+
+<!-- Please describe clearly and concisely what the bug is. -->
+
+## Steps to reproduce
+
+<!-- Please provide detailed steps on how to reproduce the bug. -->
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+## Screenshots
+
+<!-- If applicable, please add screenshots to help explain your problem. -->
+
+## Additional Context
+
+<!-- Please complete the following information. -->
+ - PHP Version: 
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Plugin Version [e.g. 22]
+ - Device: [e.g. iPhone6]
+
+<!-- Please add any additional information about the bug. -->
```

Comment:
Perhaps we should request that they include a dump of their Site Health info here as well?